include/aos/iam/certhandler.hpp

```diff
@@ -24,11 +190,82 @@ namespace certhandler {
 class CertHandler {
 public:
     /**
-     * Creates key.
+     * RegisterModule registers module.
```

Comment:
In go it was required to put method or var name in first place in comment. For oxygen it is not required. Should be just: `Registers module`. For other methods as well.